packages/bruno-electron/src/utils/proxy-util.js

```
@@ -84,6 +83,27 @@ class PatchedHttpsProxyAgent extends HttpsProxyAgent {
+          const { proxy: proxyUri, httpProxyAgentOptions } = options;
+          super(proxyUri, httpProxyAgentOptions);
+          this.timeline = Array.isArray(timeline) ? timeline : [];
+          // Log the proxy details
+          this.timeline.push({
+            timestamp: new Date(),
+            type: 'info',
+            message: `Using proxy: ${proxyUri}`
+          });
+        }
+      }
+    }
```

Biome (2.1.2)

[error] This constructor has code paths that return without calling `super()`.

If this is intentional, add an explicit throw statement in unsupported paths.

(lint/correctness/noUnreachableSuper)